hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/jdbc/join/JoinPredicateProcessingResult.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2023 Hazelcast Inc.
+ *
+ * Licensed under the Hazelcast Community License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://hazelcast.com/hazelcast-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.sql.impl.connector.jdbc.join;
+
+import com.hazelcast.sql.impl.row.JetSqlRow;
+
+public class JoinPredicateProcessingResult {
```

Comment:
Does it make sense to make this class an inner class of `JoinPredicateScanRowMapper`?

Reply:
Maybe but I don't like inner classes. Personal preference let's say :-D